cumulusci/tasks/metadeploy.py

```diff
@@ -19,6 +19,8 @@ def _call_api(self, method, path, collect_pages=False, **kwargs):
         results = []
         while next_url is not None:
             response = self.api.request(method, next_url, **kwargs)
+            if response.status_code == 400:
+                raise requests.exceptions.HTTPError(response.content)
```

Comment:
Why? `response.raise_for_status` already does this, with the response accessible as err.response

Reply:
Oh, to make sure the response content is shown in the log.